docker/Dockerfile

```diff
@@ -80,7 +80,7 @@ RUN wget -q https://github.com/protocolbuffers/protobuf/releases/download/v3.11.
     make && \
     make install && \
     ldconfig && \
-    rm -rf /tmp/protobuf-3.11.4
+    rm -rf /tmp/protobuf-3.11.4 /tmp/protobuf-all-3.11.4.tar.gz
```

Comment:
How come we're not deleting this tarball anymore?

Reply:
> How come we're not deleting this tarball anymore?

We previously didn't delete the tarball and we should right?

Reply:
....duh. Sorry. I should not do code reviews first thing in the morning :)